pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -146,40 +147,56 @@ type PlayerSelector struct {
 }
 
 // CounterSelector is the filter options for a GameServer based on the count and/or available capacity.
-// 0 for MaxCount or MaxAvailable means unlimited maximum. Default for all fields: 0
 type CounterSelector struct {
-	MinCount     int64 `json:"minCount"`
-	MaxCount     int64 `json:"maxCount"`
+	// MinCount is the minimum current value. Defaults to 0.
+	// +optional
+	MinCount int64 `json:"minCount"`
+	// MaxCount is the maximum current value. Defaults to 0, which translates as max(in64).
```

Comment:
(not for this release) This 0 as `max(int64)` feels a bit confusing to me. Should we make a default for `MaxCount` (and the others in this pattern) to be `max(int64)`, so it shows up in the CRD values as that at inspection time -- which I would argue is more intuitive to the end user?

Reply:
max(int64) = 9223372036854775807 which also seems non-intuitive. We could probably default to a round number, although I don't know what would make the most sense.

Reply:
Yeah, that's a good point - that massive a number is ... massive 😄 okay, let's leave as is and see what feedback ends up being.